appveyor.yml

```diff
@@ -15,6 +16,11 @@ skip_tags: true
 
 clone_folder: c:\projects\habitat
 
+environment:
+  matrix:
+    - hab_build_action: "test;build"
```

Comment:
yeah, it's WAAAY slower.  
* sharing a workspace for each build step allows single pass compilation of dependencies
* if we don't want the build matrix to progress serially, there's a dollar cost ($100/mo just to get two concurrent jobs).